src/utils/DataSourceHelper/DataSourceHelper.ts

```diff
@@ -0,0 +1,112 @@
+import "./DataSourceHelper.scss";
+
+interface ConstraintStore {
+    [widgetId: string]: string | HybridConstraint;
+}
+
+interface Version {
+    major: number; minor: number; path: number;
```

Comment:
Why not have these the proper way? Each on its own line.
Also it's `patch` not `path`.

Reply:
Done